### ISSUE TITLE
Cannot read property 'length' of null

### ISSUE THREAD

**Issue by nmara** [avocode/avocode#1952]
---
* **Version**: 2.21.0
* **Web/Desktop**: Desktop
* **Platform**: linux
* **Avocode UID**: 295939
---
```
TypeError: Cannot read property 'length' of null
    at Function.value (file:///opt/Avocode/resources/app.asar/dist/avocode.min.js:1:216683)
    at t.value (file:///opt/Avocode/resources/app.asar/dist/avocode.min.js:1:157392)
    at ReactCompositeComponentWrapper._renderValidatedComponentWithoutOwnerOrContext (/opt/Avocode/resources/app.asar/node_modules/react-dom/lib/ReactCompositeComponent.js:799:30)
    at ReactCompositeComponentWrapper._renderValidatedComponent (/opt/Avocode/resources/app.asar/node_modules/react-dom/lib/ReactCompositeComponent.js:822:32)
    at ReactCompositeComponentWrapper.performInitialMount (/opt/Avocode/resources/app.asar/node_modules/react-dom/lib/ReactCompositeComponent.js:362:30)
    at ReactCompositeComponentWrapper.mountComponent (/opt/Avocode/resources/app.asar/node_modules/react-dom/lib/ReactCompositeComponent.js:258:21)
    at Object.mountComponent (/opt/Avocode/resources/app.asar/node_modules/react-dom/lib/ReactReconciler.js:46:35)
    at ReactDOMComp
```

**Comment by jakubzitny** [comment]
Hi @nmara, thanks for reporting this problem, we can see it in our bug tracking system, we'll try to fix as soon as possible. I guess this happened only once, so you can use Avocode normally, right?

**Comment by jakubzitny** [comment]
Hi, this problem should be released in latest Avocode 2.22.0. Please update and let us know if it's still happening.
